client/components/TestQueue/queries.js

```diff
@@ -70,6 +70,7 @@ export const TEST_QUEUE_PAGE_QUERY = gql`
             }
             draftTestPlanRuns {
                 id
+                initiatedByAutomation
```

Comment:
Would it be possible to use this value in the reports page as well? Currently, even if test results were collected by a bot, they are indistinguishable from manually-collected results in the run history section.

<img width="1420" alt="Screenshot 2023-12-18 at 4 33 09 PM" src="https://github.com/w3c/aria-at-app/assets/5255858/36694bb0-4cd7-47d6-b275-b3bc1c9f8972">

Reply:
I agree but I think this one might need designs as well so it may be part of a v2. I'll compile all of your design related feedback into something that can be referenced when the next round of designs are being drawn up.

Reply:
Archived this feedback in [this comment](https://bocoup.slack.com/archives/CB9ESG2AF/p1703199005107479) for possible use on further design cycles.